contracts/interface/IClearingHouseConfig.sol

```diff
@@ -1,7 +1,27 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 pragma solidity 0.7.6;
 
-interface IClearingHouseConfig {
+interface IClearingHouseConfigEvent {
+    event LiquidationPenaltyRatioChanged(uint24 liquidationPenaltyRatio);
+
+    event PartialCloseRatioChanged(uint24 partialCloseRatio);
+
+    event TwapIntervalChanged(uint32 twapInterval);
```

Comment:
One thing worth noticing is that the param used to be `uint256 twapInterval` but it should be `uint32` so I changed it here.
Not 100% sure but I think this won't break anything but will love to be corrected if this isn't true!

Reply:
Change type of param will derive different topic of event.
Need to check if it's used in subgraph.